realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -2322,16 +2057,12 @@ private OsResults lazyFindAll() {
 
     /**
      * Create a text-based predicate using the Realm Query Language. This predicate can be combined
-     * with other raw or type safe predicates, it accepts Realm primitives or Realm objects as arguments.
+     * with other raw or type safe predicates, it accepts Realm values as arguments.
      * <p>
      * Class and property names used in the raw predicate can be either the names defined in the
      * Realm Model classes or the internal names defined using the {@link io.realm.annotations.RealmClass}
      * or {@link io.realm.annotations.RealmField} annotations.
-     * <p>
```

Comment:
Any reason to remove this?
We should probably also add something about neededing to escape field names with spaces?

Reply:
Shouldn't have removed. Added a comment on escaping field and class names.